README.md

```diff
@@ -0,0 +1,35 @@
+# Try: Simplified Error Handling in Go
+
+[![GoDev](https://img.shields.io/static/v1?label=godev&message=reference&color=00add8)][godev]
+[![Build Status](https://github.com/dsnet/try/actions/workflows/test.yml/badge.svg?branch=master)][actions]
+
+This module reduces the syntactic cost of error handling in Go.
+
+Example usage:
+
+```go
+func main() {
+    defer try.F(log.Fatal)
+    b := try.E1(os.ReadFile(...))
+    var v any
+    try.E(json.Unmarshal(b, &v))
+    ...
+}
+```
+
```

Comment:
(optional) It'd be nice to add more examples here, since it'll be peoples' first impression of what they can do with it.

Reply:
PTAL.

Reply:
LGTM